ext/pdo_pgsql/pgsql_statement.c

```diff
@@ -727,6 +727,7 @@ static int pgsql_stmt_get_attr(pdo_stmt_t *stmt, zend_long attr, zval *val)
 #endif
 
 		default:
+			(void)S;
```

Comment:
makes sense, we could have created the var only for the case above but one day we might add more attributes to fetch.